Marlin/src/lcd/menu/menu_media.cpp

```diff
@@ -126,17 +126,18 @@ void menu_media_filelist() {
   else if (card.isMounted())
     ACTION_ITEM_F(F(LCD_STR_FOLDER " .."), lcd_sd_updir);
 
-  if (ui.should_draw()) for (int16_t i = 0; i < fileCnt; i++) {
-    if (_menuLineNr == _thisItemNr) {
-      card.selectFileByIndexSorted(i);
-      if (card.flag.filenameIsDir)
-        MENU_ITEM(sdfolder, MSG_MEDIA_MENU, card);
+  if (ui.should_draw())
+    for (int16_t i = 0; i < fileCnt; i++) {
+      if (_menuLineNr == _thisItemNr) {
+        card.selectFileByIndexSorted(i);
+        if (card.flag.filenameIsDir)
+          MENU_ITEM(sdfolder, MSG_MEDIA_MENU, card);
+        else
+          MENU_ITEM(sdfile, MSG_MEDIA_MENU, card);
+      }
       else
-        MENU_ITEM(sdfile, MSG_MEDIA_MENU, card);
+        SKIP_ITEM();
```

Comment:
What do you think of inverting the condition, so the shorter part of the `if` is placed at the beginning.
In that case, it would be easier to tell when the item is skipped.

Reply:
The same as above - just incidentally saw this "if".